src/firefly/java/edu/caltech/ipac/util/VoTableUtil.java

```diff
@@ -45,6 +55,134 @@
         return groups.toArray(new DataGroup[groups.size()]);
     }
 
+    private enum MetaInfo {
+        INDEX("Index", "Index", Integer.class, "table index", false),
+        TABLE("Table", "Table", String.class, "table name", false),
+        TYPE("Type", "Type", String.class, "table type", false),
+        NAME("Name", "Name", String.class, "table name", true),
+        ROW("Rows", "Total Rows", Long.class, "total rows", true),
+        COLUMN("Columns", "Total Columns", Integer.class, "total columns", true);
+
+        String keyName;
+        String title;
+        Class  metaClass;
+        String description;
+        boolean bRowInfo;
+
+        MetaInfo(String key, String title, Class c, String des, boolean rowInfo) {
+            this.keyName = key;
+            this.title = title;
+            this.metaClass = c;
+            this.description = des;
+            this.bRowInfo = rowInfo;
+        }
+
+        List<Object> getInfo() {
+            return Arrays.asList(keyName, title, metaClass, description);
+        }
+
+        String getKey() {
+            return keyName;
+        }
+
+        String getTitle() {
+            return title;
+        }
+
+        Class getMetaClass() {
+            return metaClass;
+        }
+
+        String getDescription() {
+            return description;
+        }
+
+        JSONObject addEntryToJson(Object value) {
+            JSONObject oneObj = new JSONObject();
+
+            oneObj.put("key", getTitle());
+            oneObj.put("value", value);
+            return oneObj;
+        }
+
+        boolean isRowInfo() { return bRowInfo; }
+
+    }
+
+    public static DataGroup voHeaderToDataGroup(String voTableFile) {
+        List<DataType> cols = new ArrayList<DataType>();
+
+        for ( MetaInfo meta : MetaInfo.values()) {    // index, name, row, column
+            if (meta.isRowInfo()) continue;
+            DataType dt = new DataType(meta.getKey(), meta.getTitle(), meta.getMetaClass());
+            dt.setShortDesc(meta.getDescription());
+            cols.add(dt);
+        }
+        DataGroup dg = new DataGroup("votable", cols);
+        String invalidMsg = "invalid votable file";
+
+        try {
+            StarTableFactory stFactory = new StarTableFactory();
+            TableSequence tseq = stFactory.makeStarTables(voTableFile, null);
```

Comment:
Same as fits images.  Is there a way to get the headers only?  
Also, if we're reading the full file anyways, why not use existing function to read it into a DataGroup?
I am also concern with using json string as parsed values.  There's no way to reuse this code.